demo/main.gd

```diff
@@ -22,6 +24,20 @@ func _ready():
 	for hand in OpenXRInterface.HAND_MAX:
 		hand_tracking_source[hand] = xr_interface.get_hand_tracking_source(hand)
 
+	if Engine.has_singleton("OpenXRFbEnvironmentDepthExtensionWrapper"):
+		await await get_tree().create_timer(1.0).timeout
```

Comment:
For some reason, the game crashes when deployed to my Quest 3 if the extension is started too early. This will obviously need to be handled better in the future.